account_chart_update/tests/test_account_chart_update.py

```diff
@@ -466,3 +466,14 @@ def test_00_matching(self):
         self.assertTrue(list(self.fp.get_external_id().values())[0])
         self.assertEqual(fp_rec.get_external_id().get(fp_id), expected_xmlid)
         wizard.unlink()
+
+    def test_01_archived_fiscal_position(self):
+        # Test wizard won't duplicate existing fiscal positions when archived
+        fiscal_position = self.env["account.fiscal.position"].search(
+            [("name", "=", self.fp_template.name), ("company_id", "=", self.company.id)]
+        )
+        fiscal_position.active = False
+        wizard = self.wizard_obj.create(self.wizard_vals)
+        wizard.action_find_records()
+        wizard.action_update_records()
+        wizard.unlink()
```

Comment:
issue: just make a final assertion here, for test completeness:

```suggestion
        wizard.unlink()
        self.assertTrue(fiscal_position.exists())
        self.assertFalse(fiscal_position.active)
```